src/app/clusters/microwave-oven-control-server/microwave-oven-control-server.cpp

```diff
@@ -318,8 +318,8 @@ void Instance::HandleSetCookingParameters(HandlerContext & ctx, const Commands::
             mSupportedWattLevels > 0,
             ChipLogError(Zcl, "Microwave Oven Control: Failed to set wattSettingIndex, count of supported watt levels is 0"));
 
-        reqWattSettingIndex = wattSettingIndex.ValueOr(mSupportedWattLevels - 1);
-        VerifyOrExit(reqWattSettingIndex <= (mSupportedWattLevels - 1), status = Status::ConstraintError;
+        reqWattSettingIndex = wattSettingIndex.ValueOr(static_cast<uint8_t>(mSupportedWattLevels - 1));
```

Comment:
Might have been clearer to do `uint8_t maxWattLevelIndex = static_cast<uint8_t>(mSupportedWattLevels - 1);` and then use that in the checks, but this is fine.